Formula/git-gui.rb

```diff
@@ -0,0 +1,50 @@
+class GitGui < Formula
+  desc "Tcl/Tk UI for the git revision control system"
+  homepage "https://git-scm.com"
+  # Note: these should be kept in sync with git.rb
+  url "https://www.kernel.org/pub/software/scm/git/git-2.25.0.tar.xz"
+  sha256 "c060291a3ffb43d7c99f4aa5c4d37d3751cf6bca683e7344ea407ea504d9a8d0"
```

Comment:
I am not sure if it's possible to use something like `Git.sha256` and `Git.url` to prevent these from become out of sync?

Reply:
This compiles
```ruby
  homepage "#{Formula["git"].homepage}"
  url "#{Formula["git"].stable.url}"
  sha256 "#{Formula["git"].stable.checksum}"
  head "#{Formula["git"].head}", :shallow => false
```

Reply:
Thanks, I've updated the PR.